ruleset.xml

```diff
@@ -1,23 +1,5 @@
 <?xml version="1.0"?>
 <ruleset name="Zend Framework Coding Standard">
     <description>Zend Framework Coding Standard</description>
-
-    <!-- display progress -->
-    <arg value="p"/>
-    <arg name="colors"/>
-
-    <!-- inherit rules from: -->
-    <rule ref="PSR2"/>
-    <rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
-    <rule ref="Generic.Formatting.SpaceAfterNot"/>
-    <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-        <properties>
-            <property name="ignoreNewlines" value="true"/>
-        </properties>
-    </rule>
-    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
-        <properties>
-            <property name="ignoreBlankLines" value="false"/>
-        </properties>
-    </rule>
+    <rule ref="vendor/zendframework/zend-coding-standard/src/ZendCodingStandard/ruleset.xml"/>
```

Comment:
How to properly include the ruleset in other projects?

1. `<rule ref="vendor/zendframework/zend-coding-standard/ruleset.xml"/>`
2. `<rule ref="vendor/zendframework/zend-coding-standard/src/ZendCodingStandard/ruleset.xml"/>`

The official ruleset.xml file has to be located inside `src/ZendCodingStandard` otherwise PHP_CodeSniffer doesn't detect the ruleset. 
Option 1 is basically a clean wrapper to load option 2. If in the future PHP_CodeSniffer changes something we can just adjust the wrapper without changing all packages. But I don't see this happening any time soon. Option 2 loads the ruleset directly.